docs/installation.md

```diff
@@ -0,0 +1,382 @@
+# Installation for production
+
+This page document how to perform a native deployement on a Debian-like environment,
+for a `udata` user with home in`/srv/udata`.
+These steps require the [system dependencies](system-dependencies.md) to be installed.
+
+# User and home dir creation
+
+We want to create a `udata` user which primary group is `udata`,
+member of the `www-data` group and having its home directory in `/srv/udata`:
+
+```shell
+$ useradd -m -d /srv/udata -G www-data udata
+```
+
+You can check the result with:
+
+```shell
+$ id udata
+uid=1001(udata) gid=1001(udata) groups=33(www-data)
+$ ls -l /srv
+total 4
+drwxr-xr-x 9 udata udata 4096 Jun 23 05:50 udata
+```
+
+You can now log into this account to install uData:
+
+```shell
+$ su - udata
+$ pwd
+/srv/udata
+```
+
+## Python and virtual environment
+
+It is recommended to work within a virtualenv to ensure proper dependencies isolation.
+If you're not familiar with that concept, read [Python Virtual Environments - a Primer][].
+
+We create a virtualenv in the udata home directory so it is activated each time
```

Comment:
Yes, it's why I create and user with a home and a root virtualenv in production, it makes it easy to work with. The trick is that the `$HOME/bin` directory is automatically added on the `$PATH`, at least on Debian ;)